go.work

```diff
@@ -0,0 +1,382 @@
+	./processor/k8sattributesprocessor
+	./processor/logstransformprocessor
+	./processor/metricsgenerationprocessor
+	./processor/metricstransformprocessor
+	./processor/probabilisticsamplerprocessor
+	./processor/redactionprocessor
+	./processor/resourcedetectionprocessor
+	./processor/resourceprocessor
+	./processor/routingprocessor
+	./processor/schemaprocessor
+	./processor/spanmetricsprocessor
+	./processor/spanprocessor
+	./processor/tailsamplingprocessor
+	./processor/transformprocessor
+	./receiver/activedirectorydsreceiver
+	./receiver/aerospikereceiver
+	./receiver/apachereceiver
+	./receiver/awscontainerinsightreceiver
+	./receiver/awsecscontainermetricsreceiver
+	./receiver/awsfirehosereceiver
+	./receiver/awsxrayreceiver
+	./receiver/azureeventhubreceiver
+	./receiver/bigipreceiver
+	./receiver/carbonreceiver
+	./receiver/chronyreceiver
+	./receiver/cloudfoundryreceiver
+	./receiver/collectdreceiver
+	./receiver/couchdbreceiver
+	./receiver/dockerstatsreceiver
+	./receiver/dotnetdiagnosticsreceiver
+	./receiver/elasticsearchreceiver
+	./receiver/expvarreceiver
+	./receiver/filelogreceiver
+	./receiver/flinkmetricsreceiver
+	./receiver/fluentforwardreceiver
+	./receiver/googlecloudpubsubreceiver
+	./receiver/googlecloudspannerreceiver
+	./receiver/hostmetricsreceiver
+	./receiver/iisreceiver
+	./receiver/influxdbreceiver
+	./receiver/jaegerreceiver
+	./receiver/jmxreceiver
+	./receiver/journaldreceiver
+	./receiver/k8sclusterreceiver
+	./receiver/k8seventsreceiver
+	./receiver/kafkametricsreceiver
+	./receiver/kafkareceiver
+	./receiver/kubeletstatsreceiver
+	./receiver/memcachedreceiver
+	./receiver/mongodbatlasreceiver
+	./receiver/mongodbreceiver
+	./receiver/mysqlreceiver
+	./receiver/nginxreceiver
+	./receiver/nsxtreceiver
+	./receiver/opencensusreceiver
+	./receiver/otlpjsonfilereceiver
+	./receiver/podmanreceiver
+	./receiver/postgresqlreceiver
+	./receiver/prometheusexecreceiver
+	./receiver/prometheusreceiver
+	./receiver/rabbitmqreceiver
+	./receiver/receivercreator
+	./receiver/redisreceiver
+	./receiver/riakreceiver
+	./receiver/saphanareceiver
+	./receiver/sapmreceiver
+	./receiver/signalfxreceiver
+	./receiver/simpleprometheusreceiver
+	./receiver/simpleprometheusreceiver/examples/federation/prom-counter
+	./receiver/skywalkingreceiver
+	./receiver/solacereceiver
+	./receiver/splunkhecreceiver
+	./receiver/sqlqueryreceiver
+	./receiver/sqlserverreceiver
+	./receiver/statsdreceiver
+	./receiver/syslogreceiver
+	./receiver/tcplogreceiver
+	./receiver/udplogreceiver
+	./receiver/vcenterreceiver
+	./receiver/wavefrontreceiver
+	./receiver/windowseventlogreceiver
+	./receiver/windowsperfcountersreceiver
+	./receiver/zipkinreceiver
+	./receiver/zookeeperreceiver
+	./testbed
+	./testbed/mockdatareceivers/mockawsxrayreceiver
+)
+
+// 	references to modules that are in this repository with their relative paths
+// so that we always build with current (latest) version of the source code.
+
+replace (
```

Comment:
I assume this would cause the release process to change a bit. If that's the case, can we have an update to it as part of this PR?

Reply:
Absolutely.